core/src/main/scala/org/http4s/blaze/channel/nio1/Selectable.scala

```diff
@@ -22,11 +22,5 @@ private trait Selectable {
     *
     * This close method represents closure under normal conditions.
     */
-  def close(): Unit
-
-  /** Close this `Selectable` and release associated resources.
-    *
-    * This close method represents closure under exceptional conditions.
-    */
-  def closeWithError(cause: Throwable): Unit
+  def close(cause: Option[Throwable]): Unit
```

Comment:
Is the scaladoc about "normal conditions" still true when there is some `cause`?